repair-service/model/src/main/java/com/epam/repair/model/Address.java

```diff
@@ -0,0 +1,137 @@
+package com.epam.repair.model;
+
+import javax.persistence.*;
+
+/**
+ * The type Address.
+ */
+@Entity
+@Table(name = "address")
+public class Address {
+
+    /**
+     * City id is the primary key.
+     */
+    @Id
+    @GeneratedValue(strategy= GenerationType.SEQUENCE,
+            generator="address_generator")
+    @SequenceGenerator(name="address_generator",
+            sequenceName="address_generator", allocationSize=1000)
+    @Column(name="address_id", updatable=false, nullable=false)
+    private Integer addressId;
+
+    @Column(name="address_number_house", length = 40, nullable=false)
+    private String houseNumber;
+
+    @Column(name="address_number_apartment", length = 40, nullable=false)
+    private String apartmentNumber;
+
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
Are you sure that `EAGER` should be here?

Reply:
Why not? I thought, if I want to get all the address data completely, I need EAGER.